spec/install/gems/resolving_spec.rb

```diff
@@ -150,6 +150,13 @@
 
       let(:ruby_requirement) { %("#{RUBY_VERSION}") }
       let(:error_message_requirement) { "~> #{RUBY_VERSION}.0" }
+      let(:error_message_platform) do
+        if Bundler.feature_flag.specific_platform?
+          " #{Bundler.local_platform}"
+        else
+          ""
+        end
+      end
```

Comment:
Only this is the incompatible behavior.
The current local platform information is shown in error message with `specific_platform` feature flag because we try platform specific version as the most priority candidate.
Is this acceptable?

Reply:
I think that's perfectly acceptable, yeah!